unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -459,6 +470,21 @@ public override void Cleanup()
 
             avatarReporterController.ReportAvatarRemoved();
         }
+        
+        public void UpdateOutOfBoundariesState(bool isInsideBoundaries)
+        {
+            if (scene.isPersistent)
```

Comment:
Could be changed to `isInsideBoundaries = scene.isPersistent`

Reply:
Actually that wouldn't work here because in this case we want to treat the scene boundaries state application as "inside boundaries" if the scene is global (isPersistent) but we don't want to set the `isInsideBoundaries` as false every time the scene is not global (what you are suggesting would have that effect)